packages/attendance/src/components/CalendarBar.js

```diff
@@ -24,7 +24,7 @@ const FormatDate = ({ date, type }) => {
   }
 };
 
-export default function CalendarBar({ view, ...props }) {
+export default function DCalendarBar({ view, ...props }) {
```

Comment:
What does `D` signify? Can we make this verbose if this is something new that we are creating?